manager/volume.go

```diff
@@ -391,6 +391,16 @@ func (m *VolumeManager) Activate(volumeName string, frontend string) (v *longhor
 		return nil, fmt.Errorf("invalid frontend %v", frontend)
 	}
 
+	_, err = m.ListBackupVolumes()
```

Comment:
Add a comment that `ListBackupVolumes` will trigger the update for LastBackup.

And is this enough? Does LastBackup updated will immediately for the engine to set `engine.Spec.RequestedBackupRestore`? It sounds like an asynchronized process.